### ISSUE TITLE
content loss

### ISSUE THREAD

**Issue by lizetian555** [xinntao/BasicSR#86]
thank you but I print the content loss,which is 0.why ?

**Comment by xinntao** [comment]
there must something wrong. The pixel loss should not be zero.

**Comment by Achhhe** [comment]
It seems that I have the similar problem. 
I change the G and train a psnr_G from scratch. But my pixel loss is very small, most are close to zero(such as 0.1, 0.08, 0.05).

**Comment by xinntao** [comment]
@Achhhe , Yes, the pixel loss is very small, usually in the range [1e-4, 1e-2]. This is a problem.

**Comment by Achhhe** [comment]
@xinntao 
Thank you for your reply.
I think the pixel loss is small because the default value of size_average in nn.L1 Loss is true. The loss will be very large when size_average=false

**Comment by xinntao** [comment]
The pixel loss is small. I cannot remember why I put `This is a problem.` in my last reply. 
The loss is not 0, but close to 0, in the range [1e-4, 1e-2].
